README.md

```diff
@@ -135,12 +135,12 @@ Klaviyo.setEmail("robin@example.com")
 ### Tracking Events
 
 The SDK also provides tools for tracking analytics events to the Klaviyo API.
-A list of common Klaviyo-defined event names is provided in [EventType](https://klaviyo.github.io/klaviyo-android-sdk/1.0.0/sdk/analytics/com.klaviyo.analytics.model/-event-type/), or
-you can use `EventType.CUSTOM("name")` for custom event names.
+A list of common Klaviyo-defined event metrics is provided in `EventMetric`, or
```

Comment:
Hm lost a git comment. 
Yeah I took the link out because we were only doing this link to documentation in one place, thought we should take a consistent approach. Maybe when we re-write the readme we can re-consider. It is just one more thing to remember at release time too.

Reply:
We also have that epic in Jira we've backlogged for a long time now where we retro'd on our readmes and had a whole plan to rewrite everything to encourage consistency between iOS and Android setup steps. Assuming we can get around to actually implementing some of that in time, maybe small decisions here aren't super super important right now.

Reply:
Exactly